src/main/scala/com/databricks/sparkdl/DeepImageFeaturizer.scala

```diff
@@ -38,6 +38,7 @@ class DeepImageFeaturizer(override val uid: String) extends Transformer with Def
 
   final val inputCol: Param[String] = new Param[String](this, "inputCol", "input column name")
   final val outputCol: Param[String] = new Param[String](this, "outputCol", "output column name")
+  final val scaleFast: Param[Boolean] = new Param[Boolean](this,"scaleFast","use fast resizing if set.")
```

Comment:
what does "fast resizing" mean?

Reply:
it's just a flag singnaling to the resizer to use the other resize method. Not sure how to name it it's not mapped to a single flag on the java side either. SCALE_FAST or SCALE_DEFAULT would work, probably a few others as well. the name is WIP